wandb/cli/cli.py

```diff
@@ -1074,6 +1075,9 @@ def launch(
     else:
         resource_args = {}
 
+    if entry_point is not None:
+        entry_point = shlex.split(entry_point)
+
```

Comment:
Maybe move this into EntryPoint.__init__() ? Making it robust to either `str` or `List[str]`

Reply:
I think we want to only support a list of strings as an entrypoint, it makes the launch config modal and config file creation more simple.